unobot.py

```diff
@@ -398,7 +405,7 @@ def render_cards_colored(cards, theme=THEME_NONE):
                 green_code = colors.GREEN
                 yellow_code = colors.ORANGE
         ret = []
-        for card in sorted(cards):
+        for card in cards:
             if card in ['W', 'WD4']:
```

Comment:
Why doesn't this use `if 'W' in card` like above? I don't even remember. (And yes, I know this and the above comment aren't even changes from this PR.)